### DISCUSSION TITLE
ViewBug in Mobile Version including a picture in Readme.md

### DISCUSSION THREAD

**Discussion by n0tskrty** [community/community#53462] [Mobile]
Is it just me? I cleared the cache. Repo is public:

https://github.com/Snoocx/GFN_Randomizr

### Here's the screen of the mobile app:
![image](https://user-images.githubusercontent.com/120705164/233747464-41ce4e96-17eb-44ea-ad17-b15eff60e320.png)

#
### Here's the screen of the Webversion using Safari:
![image](https://user-images.githubusercontent.com/120705164/233747472-cbedd531-64e8-428a-9b02-3ea6cdc0448f.png)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.96.0 (88967961)
Device: iPhone15,3 (iOS 16.2)
</details>

**Comment by BenjaminMichaelis** [comment]
Seems similar to https://github.com/orgs/community/discussions/38049

**Comment by taki-on** [comment]
Thank you for reaching out to us. I will file this as a bug and investigate. We have some similar bug reports we got and we have our eyes on them. Thank you!
